dist/css/charts/row.css

```diff
@@ -22,3 +22,10 @@
 
 .row-chart .percentage-label {
   fill: #666A73; }
+
+.row-chart .view-more-group .view-more-background {
+  fill: aqua; }
```

Comment:
This is able to be overwritten correct?

Reply:
yes, correct. that's only if someone decides to import the css.  I had to have that in here so the demo would work.